src/app/sharedUI/screens/Artwork/ArtworkContent/ArtworkContent.tsx

```diff
@@ -28,6 +31,9 @@ export const ArtworkContent = ({ slug }: { slug: string }) => {
   const [isScrollEnabled, setIsScrollEnabled] = useState(false)
   const color = useColor()
   const space = useSpace()
+  const { width, height } = Dimensions.get("window")
+  const [isWebViewVisible, toggleIsWebViewVisible] = useState(false)
```

Comment:
```suggestion
  const [isWebViewVisible, setIsWebViewVisible] = useState(false)
```
As it's a setState function. I think its better to follow react pattern.